docs/faq.md

```diff
@@ -30,22 +30,25 @@ Specifically, we remove any genes that are not measured in every sample in a dat
 When using quantile normalization, the expression value a gene is assigned in a particular sample depends on the _rank_ of that gene.
 If a user download different datasets, which may have different numbers of genes, it's possible then that the same gene in the same sample would have a different expression value between them.
 
-#### Why do I get a limited number of genes back when I aggregate samples from different platforms?
+#### Why do I get a limited number of genes back when I aggregate samples from different experiments?
 
-Different platforms will often measure different sets of genes.
-These differences can be particularly pronounced when comparing older microarray platforms to more recent platforms.
 When aggregating samples, we retain _only_ the genes present in _every sample_.
-If the dataset delivered to you has fewer genes than you were expecting for that organism, it could be the result of combining multiple platforms or it may be from an older microarray platform.
+Different microarray platforms will often measure different sets of genes. 
+RNA-seq samples in refine.bio can be quantified using different transcriptome indices, which may be built using different Ensembl releases and therefore include different sets of genes.
+Thus, when aggregating samples from multiple experiments, it is not uncommon for some genes measured on a platform to be dropped.
```

Comment:
I reversed platform/experiment here for what I think is improved clarity. I could be wrong.
```suggestion
Thus, when aggregating samples from multiple platforms, it is not uncommon for some genes measured in individual experiments to be dropped.
```